lib/broccoli/angular2-app.js

```diff
@@ -18,7 +18,17 @@ function Angular2App(defaults, options, additionalPaths) {
 }
 
 Angular2App.prototype.toTree = function() {
-    var sourceTree = 'src';
+    var sourceDir = 'src';
+
+    var sourceTree = new Funnel('src', {
+      destDir: 'src'
+    });
+
+    var typingsTree = new Funnel('typings', {
+      include: ['browser.d.ts', 'browser/**'],
+      destDir: 'typings'
```

Comment:
no. it's available at tmp/XXX/typings during compilation but it doesn't make it into the final output.